test/validation/api/classification/odp_classification_common.c

```diff
@@ -358,6 +359,9 @@ odp_packet_t create_packet(cls_packet_info_t pkt_info)
 		tcp->src_port = odp_cpu_to_be_16(CLS_DEFAULT_SPORT);
 		tcp->dst_port = odp_cpu_to_be_16(CLS_DEFAULT_DPORT);
 		tcp->hl = ODPH_TCPHDR_LEN / 4;
+		tcp->ack = 1;
+		tcp->fin = 0;
+		tcp->syn = 0;
```

Comment:
Could you please also initialize the other currently uninitialized fields of the TCP header. It is still possible to get an invalid TCP header out of this if the other fields (e.g. reserved bits that should be zero) happen to get certain values.